arbitrum-docs/arbitrum-ethereum-differences.mdx

```diff
@@ -44,3 +37,66 @@ Of particular note is the [ArbAddressTable](./arbos/precompiles#ArbAddressTable)
 ## Solidity
 
 You can deploy Solidity contracts onto Arbitrum just like you do Ethereum; there are only a few minor differences in behavior. See [Solidity Support](./solidity-support.mdx) for details.
+
+## JSON-RPC Comparisons
+
+This section covers the differences in response body fields you'll find when using the Arbitrum JSON-RPC vs Ethereum mainnet. 
+
+:::info
+
+Comprehensive documentation on all generally available JSON-RPC methods for Ethereum [can be found at ethereum.org](https://ethereum.org/en/developers/docs/apis/json-rpc/). As Arbitrum has `go-ethereum` at its core, most of the documented methods there can be used with no modifications.
+
+:::
+
+### Blocks
+
+When calling `eth_getBlockByHash` or `eth_getBlockByNumber`, Arbitrum includes a few additional fields and leverages some existing fields in different ways than mainnet Ethereum.
+
+#### Additional Fields
+
+- `l1BlockNumber`: An approximate L1 block number that occurred before this L2 block. See [Block Numbers and Time](time) for more info.
+- `sendCount`: The number of L2-to-L1 messages since Nitro genesis
+- `sendRoot`: The Merkle root of the Outbox tree state
+
+#### Existing Fields With Different Behavior
+
+- `extraData`: This field is equivalent to `sendRoot`
+- `mixHash`: First 8 bytes is equivalent to `sendCount`, second 8 bytes is equivalent to `l1BlockNumber`
+- `difficulty`: Fixed at `0x1`
+- `gasLimit`: Value is fixed at `0x4000000000000`, but it's important to note that Arbitrum currently has a 32M gas limit per block
+
+See full list of [mainnet block fields](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getblockbyhash) for additional reference.
+
+### Transactions
+
+#### Transaction Types
+
+In addition to the [three transaction types](https://ethereum.org/en/developers/docs/transactions/#types-of-transactions) currently supported on mainnet, Arbitrum adds additional types listed below and [documented in full detail here](arbos/geth#transaction-types).
+
+On calls that return custom Arbitrum type transactions, such as [`eth_getTransactionByHash`](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_gettransactionbyhash), the `type` field will reflect the custom codes where applicable. 
+
+- `100` - `ArbitrumDepositTxType`: A deposit of ETH from L1 to L2 via the Arbitrum bridge
+- `101` - `ArbitrumUnsignedTxType`: An L1 user can use to call an L2 contract via the bridge
+- `102` - `ArbitrumContractTxType`: Allows an L1 contract to call an L2 contract method via the bridge
+- `104` - `ArbitrumRetryTxType`: Used to [redeem a retryable ticket](arbos/l1-to-l2-messaging#redeeming-a-retryable) on L2, which finalizes a L2 to L1 message that failed to redeem automatically (usually due to low gas)
+- `105` - `ArbitrumSubmitRetryableTxType`: Retryable tickets are [submitted via the L1 bridge](/arbos/l1-to-l2-messaging#submitting-a-retryable) and allow arbitrary L1 to L2 messages to be created and executed on L2
```

Comment:
Something to think about (prob more generally than in this PR, but maybe nothing actionable here) is to distinguish between L1 to L2 messages generally and retryables in particular. I.e., "L1 to L2 message" could be an L2 message created b an L1 contract and sent directly to the slow inbox; what distinguishes retryables is submision and execution happen separately

Reply:
Good call out, yeah, definitely something that would be better to clearly distinguish in other places in the docs